.ci/unit_test

```diff
@@ -28,31 +28,76 @@ test_with_coverage() {
   local output_dir=test/output
   local coverprofile_file=coverprofile.out
   mkdir -p "${output_dir}"
+
   ginkgo "${GINKGO_COMMON_FLAGS[@]}" -gcflags=all=-d=checkptr=0 --coverprofile "${coverprofile_file}" -covermode=set -outputdir "${output_dir}" ${TEST_PACKAGES}
   sed -i='' '/mode: set/d' "${output_dir}/${coverprofile_file}"
   { echo "mode: set"; cat "${output_dir}/${coverprofile_file}"; } > "${output_dir}/${coverprofile_file}.temp"
   mv "${output_dir}/${coverprofile_file}.temp" "${output_dir}/${coverprofile_file}"
+
   go tool cover -func "${output_dir}/${coverprofile_file}"
 }
 
 ################################################################################
 
 # To run a specific package, run TEST_PACKAGES=<PATH_TO_PACKAGE> make test
-TEST_PACKAGES="${TEST_PACKAGES:-"cmd pkg"}"
+# If the TEST_PACKAGES is not set, then define a list of packages to run.
+TEST_PACKAGES="${TEST_PACKAGES:-"
+./pkg/backoff \
+./pkg/compactor \
+./pkg/defragmentor \
+./pkg/health/heartbeat \
+./pkg/health/membergarbagecollector \
+./pkg/initializer/validator \
+./pkg/leaderelection \
+./pkg/member \
+./pkg/metrics \
+./pkg/miscellaneous \
+./pkg/server \
+./pkg/snapshot/copier \
+./pkg/snapshot/restorer \
+./pkg/snapshot/snapshotter \
```

Comment:
Multiple packages can be run using `./pkg/snapshot/...` instead of specifying the individual packages.

Reply:
Done. Thanks